### ISSUE TITLE
DimSwitch example compilation fails on Ubuntu 

### ISSUE THREAD

**Issue by scamper07** [sinricpro/esp8266-esp32-sdk#251]
Thanks for the awesome service.

Found one issue.
Installed the latest version (2.9.15) in Arduino using Library Manager (Ubuntu 20.04) and tried to compile DimSwitch. 
It fails with 
```
/home/pre/Arduino/libraries/SinricPro/src/SinricProDimSwitch.h:12:10: fatal error: Capabilities/PowerstateController.h: No such file or directory
 #include "Capabilities/PowerstateController.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
exit status 1
Error compiling for board ESP32 Dev Module.
```

Capabilities/PowerstateController.h should be changed to **Capabilities/PowerStateController.h** in SinricProDimSwitch.h
Tested the change and now compilation works

**Comment by sivar2311** [comment]
Thanks for pointing that out.
This is because the filename case sensivitiy in linux :-/
I will fix this in the next release 2.9.17 - current release is 2.9.16

**Comment by sivar2311** [comment]
Hi @scamper07!
The fix is now in the master branch.
It will be automatically included in the next release 2.9.17.
Therefore, I am closing this issue.

Thank you very much for your contribution!
